misc/bazel/test.bzl

```diff
@@ -0,0 +1,38 @@
+def _test_script_impl(ctx):
+    output = ctx.actions.declare_file("%s.py" % ctx.label.name)
+    codeql_cli_path = ctx.toolchains["//:toolchain_type"].codeql_cli.path
+    ctx.actions.expand_template(
+        template = ctx.file._template,
+        output = output,
+        substitutions = {
+            "{codeql_cli_path}": codeql_cli_path,
+            "{test_sources}": str([f.path for f in ctx.files.srcs]),
+        },
+    )
+    return DefaultInfo(
+        files = depset([output]),
+    )
+
+_test_script = rule(
+    implementation = _test_script_impl,
+    attrs = {
+        "srcs": attr.label_list(allow_files = True),
+        "_template": attr.label(default = "//misc/bazel:test.template.py", allow_single_file = True),
+    },
+    toolchains = ["//:toolchain_type"],
+)
+
+def codeql_test(*, name, srcs, deps):
```

Comment:
The same here - I don't think this is used anywhere yet, so it shouldn't go in the tree yet.

Reply:
yep, I missed to do a proper git stash here